pom.xml

```diff
@@ -26,8 +26,8 @@
 
         <!-- Project Dependencies -->
         <java.version>11</java.version>
-        <pi4j.version>2.0-SNAPSHOT</pi4j.version>
-        <slf4j.version>2.0.0-alpha1</slf4j.version>
+        <pi4j.version>2.0</pi4j.version>
+        <slf4j.version>1.7.32</slf4j.version>
         <picocli.version>4.6.1</picocli.version>
         <junit.version>5.7.1</junit.version>
```

Comment:
<junit.version>5.7.2</junit.version>

Reply:
ah didn't see the change in junit yet, any ideas about changes in the new version?

Reply:
only  a bug fix release, I think